packages/trader/src/Modules/CFD/Containers/cfd-reset-password-modal.tsx

```diff
@@ -54,16 +57,22 @@ const CFDResetPasswordModal = ({
     platform,
     setCFDPasswordResetModal,
     history,
-}) => {
-    const [state, setState] = React.useState({
+}: TCFDResetPasswordModal) => {
+    const [state, setState] = React.useState<{
+        error_code: string | number | undefined;
+        has_error: boolean;
+        error_message: string | undefined;
+        is_finished: boolean;
+        changed_password_type: string;
+    }>({
         error_code: undefined,
         has_error: false,
         error_message: undefined,
         is_finished: false,
         changed_password_type: '',
     });
 
-    const renderErrorBox = error => {
+    const renderErrorBox = (error: { code: number | string; message: string }) => {
```

Comment:
since we have type TError in props.types.ts, I suggest the following:
```suggestion
    const renderErrorBox = (error: TError) => {
```